basic/exercise1/script.js

```diff
@@ -1,30 +1,19 @@
 'use strict';
-function formatDate(date) {
+function formatAsTwoDigits(timeUnit) {
+    return (timeUnit < 10) ? '0' + timeUnit : timeUnit;
+};
 
+function formatDate(date) {
     var days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
     var weekDays = `${days[date.getDay()]}`;
     var hours = date.getHours();
     var ampm = hours >= 12 ? 'PM' : 'AM';
     hours = hours % 12;
-    if (hours < 10) {
-        hours = "0" + hours;
-    };
-
-    var minutes = date.getMinutes();
-    if (minutes < 10) {
-        minutes = "0" + minutes;
-    };
-
-    var seconds = date.getSeconds();
-    if (seconds < 10) {
-        seconds = "0" + seconds;
-    };
-
-    var time = `${weekDays} ${hours} ${ampm} ${minutes}: ${seconds}`;
+    var minutes = formatAsTwoDigits(date.getMinutes());
+    var seconds = formatAsTwoDigits(date.getSeconds());
+    var time = `${weekDays} ${formatAsTwoDigits(hours)} ${ampm} ${minutes}: ${seconds}`;
     return time;
-
 };
```

Comment:
Please add an empty line after formatDate function definition to separate it from the rest of the code.